File: consumer-java/src/main/java/consumer/model/LeadHistory.java
package consumer.model;

import lombok.*;
import com.fasterxml.jackson.annotation.JsonInclude;

@JsonInclude(JsonInclude.Include.NON_NULL)
@Getter @Setter
public class LeadHistory {

    private String status;
    private String createdBy;
    private String date;
    private String comment;

    private String action;

    @Override
    public String toString() {
        return "LeadHistory{" +
                "status='" + status + '\'' +
                ", createdBy='" + createdBy + '\'' +
                ", date='" + date + '\'' +
                ", comment='" + comment + '\'' +
                '}';
    }
}

File: consumer-python/model/LeadHistory.py
class LeadHistory:
    def __init__(self):
        pass

    status = ""
    createdBy = ""
    date = ""
    comment = ""
    action = ""

    def __str__(self):
        return "LeadHistory{" \
               "status='" + self.status + \
               ", createdBy='" + self.createdBy + \
               ", date='" + self.date + \
               ", comment='" + self.comment + \
               '}'